src/devices/Ili9341/Ili9341Imaging.cs

```diff
@@ -6,6 +6,7 @@
 using System.Drawing.Imaging;
 using System.Runtime.InteropServices;
 
+#pragma warning disable CA1416 // Temporarily, will be removed after binding is updated
```

Comment:
please remove the comment and replace with link to new issue (unless it's already in the other PR)

Reply:
same comment everywhere where applies

Reply:
That's part of the other PR.